addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml

```diff
@@ -28,7 +28,85 @@
 			</parameter>
 		</config-description>
 	</thing-type>
+	
+	<thing-type id="ipAvr2014">
+        <label>Pioneer AVR over IP 2014</label>
+        <description>Control a 2014 Pioneer AVR over IP </description>
 
+        <channel-groups>
+            <channel-group typeId="displayInformationGroup" id="displayInformation" />
+            <channel-group typeId="zone1Controls2014" id="zone1" />
```

Comment:
It does not make sense to define several channel-group-types for the zones. Instead, you should have a single "zoneControls2014" channel group type and use that for all 4 channel groups here.

Reply:
ok. but how would i manage to give each of those 4 groups an individual label?

Reply:
Just like given in https://www.eclipse.org/smarthome/documentation/development/bindings/xml-reference.html#xml-structure-for-thing-descriptions:
```
      <channel-group id="channelGroupID" typeId="channelGroupTypeID">
        <label>String</label>
        <description>String</description>
      </channel-group>
```

Reply:
that was what i was looking for. Reworked that again. But as it is not just the label that's different but also the InputSource List for three different kind of zones i was not able to use one group for all channel groups. If i am not mistaken one can not nest item-groups.